src/DataGrid.tsx

```diff
@@ -456,6 +482,7 @@ function DataGrid<R, SR, K extends Key>(
         newColumnWidths.set(column.key, width);
         return newColumnWidths;
       });
+      setFlexColumnWidths(new Map());
```

Comment:
Re-calculated flex-column widths on resize. This is to keep the existing behavior. Ideally this is only needed if grid shows all the columns otherwise there is no need to re-calculate the widths again